mysite/lms/views.py

```diff
@@ -73,6 +73,10 @@ def get(self, request, course_id):
             request, 'lms/course_page.html',
             dict(
                 course=course,
+                course_author=(
```

Comment:
Yes and no.
We can do it in templates.
What do you think about templatetag to render user name using custom logic?